packages/nocodb-sdk/src/lib/sqlUi/MssqlUi.ts

```diff
@@ -138,9 +138,9 @@ export class MssqlUi {
     ];
   }
 
-  static getNewColumn(suffix) {
+  static getNewColumn(suffix: number, uidt: UITypes) {
```

Comment:
1. maybe call it as `columnNameSuffix` instead of `suffix` 
2. here it could be string as well. I'd suggest you always return string in `generateUniqueColumnSuffix` since you use it in a string as well.

(apply to all)